datafusion/core/tests/sqllogictests/src/engines/postgres/mod.rs

```diff
@@ -188,8 +187,12 @@ fn no_quotes(t: &str) -> &str {
 
 /// Given a file name like pg_compat_foo.slt
 /// return a schema name
-fn schema_name(file_name: &str) -> &str {
-    file_name
+fn schema_name(relative_path: &Path) -> &str {
+    relative_path
+        .file_name()
+        .unwrap()
```

Comment:
in general I think it is nice to avoid the `unwrap()` which panic's but in this case it would happen if the filenames were non utf8 -- so I think it is ok here

Reply:
I updated the logic to only keep Ascii characters https://github.com/apache/arrow-datafusion/pull/5033/commits/93f4ffde681e73a2e8ac0ba6269a7076ef5de236. I had issues with having numbers in postgres schema, so Ascii chars should be the safest option.